src/components/advanced-color-picker/advanced-color-picker.stories.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  text,
+  object,
+  withKnobs
+} from '@storybook/addon-knobs';
```

Comment:
Should be written in CSF format 
https://storybook.js.org/docs/formats/component-story-format/

Reply:
My bad, didnt notice that it's alread in CSF x)
Get rid of `State` and `Store` though, use hooks in `Basic` as you would in any other component

Reply:
updated, thanks for recommendation!